postcss.config.js

```diff
@@ -0,0 +1,7 @@
+/* eslint-disable global-require,import/no-extraneous-dependencies */
```

Comment:
Each project using this package will need to create the `postcss.config.js` file in its root, ideally extending from this root file.